src/foam/u2/dialog/ApplicationPopup.js

```diff
@@ -174,6 +186,7 @@ foam.CLASS({
           .style({ 'background-color': this.isStyled ? this.backgroundColor : ''})
           .start()
             .show(this.showActions$)
+            .enableClass('showBorder', this.progressMax$.not())
```

Comment:
More efficient to do:
.enableClass('showBorder', this.progressMax$, true)
third arg negates.